plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -36,9 +36,11 @@ import (
 )
 
 var (
-	addTfAction     = "traceflow/addTf"
-	addLiveTfAction = "traceflow/addLiveTf"
-	showGraphAction = "traceflow/showGraphAction"
+	addTfAction         = "traceflow/addTf"
+	addLiveTfAction     = "traceflow/addLiveTf"
+	showGraphAction     = "traceflow/showGraphAction"
+	runTraceAgain	    = "traceflow/runTagain"
+	mostRecentTraceflow = &crdv1alpha1.Traceflow{}
```

Comment:
shouldn't we initialized it to `nil` and display an error if someone tries to run a Traceflow again without first running a normal Traceflow?

another question: since this is mutable global state for the plugin, do we need to protect it with a mutex or is it always accessed from a single goroutine? @mengdie-song do you know?

Reply:
I cannot initialize it to null right there, but I can check if the structure is still empty before returning an error. The variable should only be accessed by traceflow.go .

Reply:
@Dhruv-J why can't you initialize it to `nil`? The following should work:
```go
var mostRecentTraceflow *crdv1alpha1.Traceflow = nil
```
Actually, the `= nil` is optional